spec/javascripts/controllers/containers/container_topology_controller_spec.js

```diff
@@ -27,4 +27,11 @@ describe('containerTopologyController', function() {
             expect($scope.kinds).not.toBe(undefined);
         });
     });
+
+    describe('kinds contain all expected kinds', function() {
+        it('in all main objects', function() {
+               expect(Object.keys($scope.kinds).length).toBeGreaterThan(7);
```

Comment:
> OK, I take it back, can you please do the whole file? Because now it's a mix of 2 a 4 spaces for indent, and that's definitely not consistent.

done